test/library/packages/Sort/RadixSort/flagsortMSB.chpl

```diff
@@ -0,0 +1,142 @@
+
+module flagsortMSB {
+  use Sort;
+  use Time;
+  use BitOps;
+  use Random;
+  config param bucketBits=8;
+  param numBuckets = (1 << bucketBits);
+  param DISTRIBUTE_BUFFER = 1;
```

Comment:
I think current unwritten convention is `param`s should be camelCased.

Also, I believe this variable can have a more descriptive name?